tests/operators/test_density_compensation_op.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for density compensation operator."""
+
+# Copyright 2024 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import torch
+from mrpro.data import DcfData
+from mrpro.operators import DensityCompensationOp
+
+from tests import RandomGenerator
+from tests.helper import dotproduct_adjointness_test
+
+
+def test_density_compensation_op_adjointness():
+    """Test density operator adjoint property."""
+    random_generator = RandomGenerator(seed=0)
+
+    ns_zyx = (2, 3, 4)
```

Comment:
In principle I like the idea of using "ns_" which I guess should mean "number of samples" but if we start using them in tests then we should (1) add it to the readme and (2) use it in our code consistently. Feel free to leave her as is and create an issue on that.

Reply:
or change to n_zyx as in the other PR. Both is fine by me.

Reply:
i switch to n_zyx for now, and if I am really bored at some point will create a PR to change all instances and add it to the readme..